src/ARCtrl/Templates/Templates.Json.fs

```diff
@@ -102,4 +102,19 @@ module Template =
                 get.Required.Field "tags" (Decode.array oaDecoder),
                 get.Required.Field "last_updated" Decode.datetimeUtc
             )
-        )
\ No newline at end of file
+        )
+
+[<AutoOpen>]
+module Extension =
+
+    type Template with
+        member this.ToJson(?spaces: int) =
```

Comment:
Stick to this naming?
https://github.com/nfdi4plants/ARCtrl/blob/14f173fea2b1efd44e26a372644800eb222f4dbd/src/ISA/ISA.Json/Study.fs#L112

Reply:
Sorry but `toString` for json sounds like a really bad idea here. I would accept changing `ofJson` to `fromJson`

Reply:
Some name unification IS necessary. Will create an ISSUE

Reply:
#206